src/Deprecated/Directory.Build.props

```
@@ -0,0 +1,7 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Does this need to be more than:
```suggestion
<Project>
```
?